src/content/en/_redirects.yaml

```
@@ -429,3 +429,56 @@ redirects:
 
 - from: /web/updates/2021/01/nic88
   to: https://developer.chrome.com/blog/new-in-chrome-88/
+
+## Redirects for Web-on-Android content migrated to DCC
+
+- from: /web/android
+  to: https://developer.chrome.com/docs/android/
```

I'd suggest moving this one to the end of the web on android redirects, and change it to:

```suggestion
- from: /web/android/...
  to: https://developer.chrome.com/docs/android/
```

That way, hard links will redirect to their appropriate place, and all other links will redirect to the dcc/docs/android/

---

Good idea! Accepted the suggestion!